### PR TITLE
Fix vibration bug for two accounts

### PR DESCRIPTION
**Pull request by yeryomenkom**
partially addresses https://github.com/wireapp/android-project/issues/173
#### APK
[Download build #11390](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11390/artifact/build/artifact/wire-dev-PR1689-11390.apk)

### CODE REVIEW

File: app/src/main/scala/com/waz/zclient/common/controllers/SoundController.scala [hunk 1]
```diff
@@ -64,9 +71,24 @@ class SoundController(implicit inj: Injector, cxt: Context) extends Injectable {
...
+    val isEnabled = (for {
+      Some(zms) <- accountsService.getZms(userId)
+      isEnabled <- zms.userPrefs.preference(UserPreferences.VibrateEnabled).signal.head
+    } yield isEnabled).recover {
+      case err =>
+        verbose(s"Error while getting vibration setting: $err")
+        false
+    }
+
+    Await.result(isEnabled, 1.second)
```
Comment (yeryomenkom) [inline app/src/main/scala/com/waz/zclient/common/controllers/SoundController.scala]: I know that this is a bad. But I do not want to refactor the "MessageNotificationsController" a lot.

**Comment by yeryomenkom** [comment]
build again

**Comment by deanrobertcook** [comment]
build again

**Comment by someoneEsle** [comment]
- [ ] Bug 1
I'm logged in with two accounts, A and B. A has vibration ON, B has vibration OFF. Incoming messages follow the right vibration preference, but calls do not. If I get a call in account B the phone vibrates.

**Comment by yeryomenkom** [comment]
build again

**Comment by yeryomenkom** [comment]
All issues fixed in https://github.com/wireapp/wire-android/pull/1732

**Pull request closed without merging**